rpc/sunrpcserver/dump_prog.go

```diff
@@ -0,0 +1,90 @@
+package sunrpcserver
+
+// GfDump is a type for GlusterFS Dump RPC program
+type GfDump genericProgram
+
+func newGfDump() *GfDump {
+	// rpc/rpc-lib/src/xdr-common.h
+	return &GfDump{
+		name:        "GF-DUMP",
+		progNum:     uint32(123451501),
```

Comment:
Using the random prognum directly will confuse people looking at this code. Please define the numbers as constants and use them instead. Name the constants using Go style camel case (if you had removed the consts to appease the linter)